### PR TITLE
Don't build Radiant when only q3map2 is requested

### LINKED ISSUE #1: Disabling something in CMake can still try to build it
Steps to reproduce: 
* `ccmake -Bbuild-test`; "c", disable netradiant, "g".
* `make -Cbuild-test`.

Notice it tries to build netradiant anyway.

### PR DESCRIPTION
**Pull request by slipher**
Fixes #1.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -118,6 +118,8 @@ if (BUILD_NETRADIANT OR BUILD_Q3MAP2)
 			CMAKE_ARGS -DGAMEPACKS_LICENSE_LIST=none
 			CMAKE_ARGS -DGAMEPACKS_NAME_LIST=${NETRADIANT_GAMEPACKS}
 			CMAKE_ARGS -DBUILD_RADIANT=ON
+		else()
+			CMAKE_ARGS -DBUILD_RADIANT=OFF
 		endif ()
 
 		if (BUILD_Q3MAP2)
```
Comment (illwieckz) [inline CMakeLists.txt:125]: maybe you take the opportunity to also add the else to q3map2

Comment (slipher) [reply CMakeLists.txt:125]: I can add it but I don't want to test it, so you need to assure me that it works :) Like q3map2 is an important subcomponent of Radiant so I wouldn't assume it works without verification.

**Comment by illwieckz** [comment]
Hmm, it looks like I just forgot to push my fixes from November 2023…

**Comment by illwieckz** [comment]
It was also needed to not download netradiant gamepacks. The master branch is now fixed.

**Pull request closed without merging**